site/using-weave/dynamically-attach-containers.md

```diff
@@ -37,6 +37,13 @@ specify `--rewrite-hosts` when running `weave attach`:
 
     host1$ weave attach --rewrite-hosts c1
 
+Weave Net will set the bridge side of the `veth` pair to enable hairpin mode when attaching a container.
```

Comment:
Default now set to true. I didn't love the `switch` statement in `attach.go`, but since the default is `true`, then we don't really need a regex, just a straight compare of `--hairpin-mode=false`